cfme/fixtures/v2v.py

```diff
@@ -19,19 +23,41 @@ def v2v_providers(request, second_provider, provider):
 
 @pytest.fixture(scope='function')
 def host_creds(request, v2v_providers):
-    """Add credentials to conversation host"""
-    provider = v2v_providers[1]
-    hosts = provider.hosts.all()
+    """Add credentials to VMware and RHV hosts."""
+    if len(v2v_providers) > 2:
+        pytest.skip("There are more than two providers in v2v_providers fixture,"
+                    "which is invalid, skipping.")
     try:
-        hosts = hosts if getattr(request, 'param', '') == 'multi-host' else hosts[0:1]
-        for host in hosts:
-            host_data, = [data for data in provider.data['hosts'] if data['name'] == host.name]
+        vmware_provider, rhv_provider = None, None
+        for provider in v2v_providers:
```

Comment:
@mshriver I was thinking about this last night, would this be better in the `v2v_providers` fixture? It can then return a keyed dict and in here we can pull assign based on the keys?

Reply:
Gotcha - discussed a bit with @kedark3, going to allow this for now.

Ultimately what I'd like to see for these tests/fixtures is use of explicit provider fixture names, and have them as parameters for any V2V fixtures that do setup on those providers. For example, `'rhv_provider'` and `'vmw_provider'` fixture names that are parameters for this fixture. Any test using this fixture should mark providers using those fixture names.